data/136/102/591/5/1361025915.geojson

```diff
@@ -27,6 +27,7 @@
     "lbl:max_zoom":18.0,
     "mz:hierarchy_label":1,
     "mz:is_current":1,
+    "mz:is_funky":0,
```

Comment:
Why are we saying it's not funky?

Reply:
There are a few of these

Reply:
I was explicit in calling out some `mz:is_funky` = 0 in this PR, but I can remove these properties (when the value is 0) if they're not useful.

Reply:
It's not wrong. If we want to note it to ourselves that this is _in fact_ not funky then we can keep it. But else general practice should be skipping these.